sound/soc/sof/topology.c

```diff
@@ -755,7 +755,8 @@ static int sof_parse_uuid_tokens(struct snd_soc_component *scomp,
 				 void *object,
```

Comment:
commit: typo "multiple identical tuples array sets" -> "multiple identical tuple array sets" ... although, typo aside, I wonder if this could be said somehow simpler. I mean "tuple array sets", there are three data structure types in a row there, wow! But yeah, if that is what it is, that's correct then.

Reply:
yes if you come up with better name I'm all in for it :) 
So we might have the ipc struct consisting of multiple tuple arrays. And as those arrays may have multiple items each, I guess they are tuples arrays?. So we have some kind of combination of tuples arrays and that's what I call a set here...

Reply:
@juimonen why not simply call it " multiple vendor tuples of the same type"? that seems to be official terminology anyway https://www.alsa-project.org/alsa-doc/alsa-lib/group__topology.html

Reply:
@ranj063 for me it sounds like all tuples would be same type? but as AFAIK they can be different types (which we push to ipc struct members, like ints, strings etc.). So to me it is more like "a set vendor tuples which make up a composite type, like a struct with possibly different typed members"

Reply:
@juimonen when I say type I dont mean the data type as in string/word/short etc but more like what type of tokens they are ie dmic_tokens/comp_tokens/pdm_tokens etc